stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageItem.kt

```diff
@@ -221,8 +221,7 @@ internal fun RowScope.DefaultMessageItemLeadingContent(
         .size(24.dp)
         .align(Alignment.Bottom)
 
-    val position = messageItem.groupPosition
-    if (!messageItem.isMine && (position == Bottom || position == None)) {
+    if (!messageItem.isMine && (messageItem.shouldShowFooter || messageItem.groupPosition == Bottom)) {
```

Comment:
This breaks some cases now - we should add `|| messageItem.groupPosition == None` at the end, after the `Bottom` check.

![image](https://user-images.githubusercontent.com/17215808/166648184-b57e16ca-f79a-4f57-86ae-4b992fb859b0.png)

An example is here: there is a date separator and the messages are split up - this causes the message to have the `None` position. Or in case the position of the message is the "sole message" (no other messages around).